app/views/proposals/details/_status.html.haml

```diff
@@ -33,3 +33,6 @@
                         = step.status.capitalize + " "
                         %span.step-timestamp
                           = step.completed_at.strftime("%m/%d/%Y %I:%M%p")
+                    - elsif step.status == "actionable" || step.status == "pending"
+                      %span.step-user-status.step-data
+                        = "Pending"
```

Comment:
There are already strings in `locales/decorators/en.yml` for this, but we don't yet have a label for "not completed". The `Step` model uses the word `outstanding` for that, so I suggest adding `outstanding: "Pending"` under `en.decorators.steps.status`